nexus/db-queries/src/db/datastore/saga.rs

```diff
@@ -450,9 +450,9 @@ mod test {
     }
 
     #[tokio::test]
-    async fn test_create_event_idempotent() {
+    async fn test_create_alert_idempotent() {
```

Comment:
We're renaming this from "test_create_event" to "test_create_alert" but it's still calling "saga_create_event"?

I think this is not an alert; I think this is a "steno" event.